hexa/core/analytics.py

```diff
@@ -0,0 +1,66 @@
+from datetime import datetime
+from uuid import uuid4
+
+from django.http import HttpRequest
+from mixpanel import BufferedConsumer, Mixpanel, MixpanelException
+from sentry_sdk import capture_exception
+from ua_parser import user_agent_parser
+
+from config.settings.base import MIX_PANEL_BUFFER_SIZE, MIX_PANEL_TOKEN
+
+mixpanel_consumer = BufferedConsumer(max_size=MIX_PANEL_BUFFER_SIZE)
+mixpanel = Mixpanel(token=MIX_PANEL_TOKEN, consumer=mixpanel_consumer)
+
+
+def track(
```

Comment:
What do you do if the analytics are not enabled for the instance?

Reply:
I'm rewriting this part to thrown an exception or return if the token is not available